SQL/tgstation_schema.sql

```diff
@@ -286,6 +286,7 @@ CREATE TABLE `player` (
   `lastadminrank` varchar(32) NOT NULL DEFAULT 'Player',
   `accountjoindate` DATE DEFAULT NULL,
   `flags` smallint(5) unsigned DEFAULT '0' NOT NULL,
+  `donator` tinyint(1) DEFAULT '0',

```

Comment:
I actually couldn't find any documentation on what the heck these `(numbers)` mean. It doesn't make sense for them to be bytes, because that's already governed by the tinyint bit, so I've no idea. I've never seen this before. Andrew halp